.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Should this run on PRs to `master`, or just once those PRs are merged?

Reply:
I often find myself wanting to use a dist for a given PR before it's merged to master. I'm not opposed to that limitation if there's a good reason

Reply:
Okay, that works for me